tasks/linux.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+action=$PT_action
+name=$PT_name
+provider=$PT_provider
+
+# example cli /opt/puppetlabs/puppet/bin/bolt  task run service::linux action=stop name=ntp --nodes localhost --modulepath /etc/puppetlabs/code/modules --password puppet --user root
+
+# high level overview
+# 3 top functions systemd() initd() upstart()
+# runs through them in order. fail fast attempt next one. if unsuccessful
+
+# psuedo-code
+# $service ()
+#   is_service_manager_installed? (capable)
+#   execute command service name and service action (do it)
+#   return the status (profit)
+
+set -e
+if [ "$action" = "start" ]; then
+  echo "{ 'Status': 'started' }"
```

Comment:
FYI JSON requires double-quoted strings, not single quoted.

Reply:
cheers @Iristyle